gradle.properties

```diff
@@ -17,4 +17,4 @@
 # under the License.
 #
 
-version=0.5.7-SNAPSHOT
+version=0.6.0-SNAPSHOT
```

Comment:
This also bumps the version up to the next minor to signal the update to Java 17. 

On 2nd thought, since we are targeting 8 at runtime, should we even worry about this version bump? Since the version numbers are still in the experimental `0` major I am ok leaving this alone. 

Wdyt?

Reply:
You are right that we don't need to bump it since it should be compatible. I'm fine with either resolution.